Vungle/VungleBannerCustomEvent.m

```diff
@@ -68,25 +62,25 @@ - (void)vungleAdDidLoad
         NSMutableDictionary *options = [NSMutableDictionary dictionary];
 
         // VunglePlayAdOptionKeyUser
-        if ([[NSUserDefaults standardUserDefaults] objectForKey:kVungleUserId]) {
-            NSString *userID = [[NSUserDefaults standardUserDefaults] objectForKey:kVungleUserId];
+        if ([self.localExtras objectForKey:kVungleUserId]) {
```

Comment:
@YueVungle - if we move to these, you may want to check the Sample Application and confirm how that should be updated for QA testing.

Reply:
I have checked the Sample App that is used by QA and added the logic about how the localExtras object will be updated by that application to pass in play options / settings. For the original NSUserDefaults accessing and saving logic, I didn't change them, they work as before, which will not affect Sample App test. I will send a PR for how the localExtras object will be updated of Sample App. :)